en/docs/install-and-setup/setup/deployment/deploying-icp-with-helm.md

```diff
@@ -0,0 +1,144 @@
+# Deploy the Integration Control Plane (ICP) on Kubernetes using Helm resources
+
+Follow the instructions below to deploy the ICP on Kubernetes (K8s) using Helm resources.
+
+## Before you begin
+    
+- Ensure you have an active [WSO2 Subscription](https://wso2.com/subscription). If you don't have a subscription, sign up for a [WSO2 Free Trial Subscription](https://wso2.com/free-trial-subscription).
```

Comment:
https://wso2.com/free-trial-subscription is redirecting to https://wso2.com/subscription/

Reply:
Asked from CS Team. Once they replied will update this.